src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
+  --selected-color: #00acdc;
+
   margin: 0;
+  padding: 0;
+  font-weight: 500;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.logoLink {
+  display: flex;
+  align-items: center;
+
+  .logo {
+    margin-left: 50px;
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  margin-right: 50px;
+
+  .navList {
+    display: flex;
+    margin: 0;
+    padding: 0;
+
+    .navListItem {
+      padding-left: 20px;
```

Comment:
i would better use margin because if you use `box-sizing: border-box`, which is recommended to use, your padding contains the element width, margin would be an external space